src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -116,6 +116,8 @@ interface State {
 export const ArtsyMapStyleURL = "mapbox://styles/artsyit/cjrb59mjb2tsq2tqxl17pfoak"
 
 const DefaultZoomLevel = 12
+const MinZoomLevel = 10
+const MaxZoomLevel = 17.5
```

Comment:
With a lot of trial and error in the Simulator this is what I found:

- 17.0 = over-clustering: many clustered-yet-distinct addresses do not expand into distinct pins
- 17.5 = best compromise: most but not all clustered-yet-distinct addresses expand to distinct pins; no disappearing shows
- 18.0 = buggy: mismatch between cluster count and pins results in "disappearing" shows as you approach max zoom